Userland/Libraries/LibMain/Main.h

```diff
@@ -13,6 +13,7 @@ namespace Main {
 struct Arguments {
     int argc {};
     char** argv {};
+    char** envp {};
```

Comment:
This creates a potentially weird situation if the program also uses setenv/putenv, or causes LibC to update `extern char** environ;` to a different value.

I'm against including this at all. If it gets added anyway, please consider the interactions with setenv/putenv, and maybe rename it to something like `original_environ`.

Reply:
Thought it would be nice to have for completeness sake, but I suppose you're right. What do you think about changing it to a reference to `environ`?

Reply:
Why do you need it though? `getenv(...)` isn't associated with main at all? Providing a nice wrapper around `getenv(...)` that returns `ErrorOr<StringView>` or `Optional<StringView>` would make sense... But I don't see the point if tying it to `Main:: Arguments`.

Reply:
Yeah, on second thought this doesn't make much sense. Especially considering one would have to be careful when using it in threaded applications.